cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -28,26 +28,269 @@ Cask Data Application Platform Release Notes
 
 New Features
 ------------
-- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+- `CDAP-20 <https://issues.cask.co/browse/CDAP-20>`__ - Removed dependency on the Guava
+  library from the ``cdap-api`` module. Applications are now free to use a Guava library
+  version of their choice.
+
+- `CDAP-3051 <https://issues.cask.co/browse/CDAP-3051>`__ - Added capability for programs to
+  perform administrative dataset operations (create, update, truncate, drop).
+
+- `CDAP-3854 <https://issues.cask.co/browse/CDAP-3854>`__ - Added the capability to
+  configure Kafka topic for logs and notifications using the ``cdap-site.xml``.
+
+- `CDAP-3980 <https://issues.cask.co/browse/CDAP-3980>`__ - MapReduce programs now support
```

Comment:
this is a little strange. MapReduce has supported this feature for a long time. We should indicate that this is MapReduce programs submitted via CDAP